openapi/data_repository_service.swagger.yaml

```diff
@@ -160,36 +159,23 @@ securityDefinitions:
     name: Authorization
     in: header
 definitions:
-  SystemMetadata:
-    type: object
-    additionalProperties: true
-    description: |-
-            OPTIONAL
-            These values are reported by the underlying object store.
-            A set of key-value pairs that represent system metadata about the object.
-  UserMetadata:
-    type: object
-    additionalProperties: true
-    description: |-
-            OPTIONAL
-            A set of key-value pairs that represent metadata provided by the uploader.
   Checksum:
     type: object
-    required: ['checksum']
+    required:
+      - checksum
     properties:
       checksum:
         type: string
-        description: |-
-          The hex-string encoded checksum for the Data.
+        description: 'The hex-string encoded checksum for the data'
       type:
         type: string
         description: |-
           The digest method used to create the checksum. If left unspecified md5
           will be assumed.
 
           possible values:
-          md5                # most blob stores provide a checksum using this
-          multipart-md5      # multipart uploads provide a specialized tag in S3
+          md5               # most blob stores provide a checksum using this
+          etag              # multipart uploads to blob stores
```

Comment:
is this a common term across clouds?  Is there a better place to define this in the docs vs. a comment?

Reply:
Not really. AFAIK the ETags specification does not specify the algorithm to be used. Service provider and users are free to choose what they use to put in the ETag header.

Typically the major cloud providers support multipart-md5. The trouble is that it depends a lot on how you have chunked your file (default is 16Mb)